components/Forms/CreateAdminPopupWindow.tsx

```diff
@@ -0,0 +1,129 @@
+import {
+  AboutEvent,
+  FormLogistics,
+  ShortFormInput,
+  LongFormInput,
+  LargeFormInput,
+  InputFlex,
+  FormInput,
+  FormLabel,
+  CreateEventForm,
+  FormHeader,
+  CreateEventContainer,
+} from '@/styles/components/event.styles';
+import {
+  SubmitButton,
+  ButtonCenter,
+} from '@/styles/components/windowFlow.styles';
+import React, { useState } from 'react';
+import PopupWindow from '@/components/PopupWindow';
+import Dayjs from 'dayjs';
+import { DatePicker, TimePicker } from 'antd';
+import { useForm } from 'react-hook-form';
+
+const CreateEventPopupWindow = ({
+  setShowPopup,
+}: {
+  setShowPopup: React.Dispatch<React.SetStateAction<boolean>>;
+}) => {
+  const { RangePicker } = DatePicker;
+
+  const [startTime, setStartTime] = useState('12:00');
+  const [endTime, setEndTime] = useState('12:00');
+
+  const { register, handleSubmit } = useForm({});
+
+  const onSubmit = (data: any) => {
+    const results = JSON.stringify({
+      firstName: data.firstName,
+      lastName: data.lastName,
+      email: data.email,
+      phone: data.phone,
+      role: data.role,
+      password: data.password,
+    });
+    console.log('Here are the results: ', results);
+    createAdmin(results);
+  };
+
+  const createAdmin = async (data: any) => {
+    console.log('Here is the data: ', data);
+    await fetch('/api/admin', {
+      method: 'POST',
+      headers: {
+        // Specify the content type in the headers
+        'Content-Type': 'application/json',
+      },
+      // Stringify the JSON body
+      body: data,
+    });
+
+    setShowPopup(false);
+  };
```

Comment:
After creating the admin, should show a message that alarms the user about it, either success or failure. You can use antd [message](https://ant.design/components/message)